VCFv4.3.tex

```diff
@@ -154,28 +154,28 @@ \subsubsection{Information field format}
 String. 
 The Number entry is an Integer that describes the number of values that
 can be included with the INFO field. For example, if the INFO field contains a
-single number, then this value should be $1$; if the INFO field describes a
-pair of numbers, then this value should be $2$ and so on. There are also
+single number, then this value must be $1$; if the INFO field describes a
+pair of numbers, then this value must be $2$ and so on. There are also
 certain special characters used to define special cases:
 
 \begin{itemize}
-  \item If the field has one value per alternate allele then this value should be `A'.
-  \item If the field has one value for each possible allele (including the reference), then this value should be `R'.
-  \item If the field has one value for each possible genotype (more relevant to the FORMAT tags) then this value should be `G'.
-  \item If the number of possible values varies, is unknown, or is unbounded, then this value should be `.'.
+  \item If the field has one value per alternate allele then this value must be `A'.
```

Comment:
Considering "special characters" introduced in the previous paragraph, that could make the flow clearer. What about making it even simpler, like the following?

> A: The field has one value per alternate allele
> R: The field has one value for each possible allele (including the reference)
> G: The field has one value for each possible genotype (more relevant to the FORMAT tags)
> . (dot): The number of possible values varies, is unknown or unbounded